src/Core/Types/Unifier.cs

```diff
@@ -581,7 +588,44 @@ private bool TryMakeFieldName(StructureField fa, StructureField fb, out string n
             return true;
         }
 
-		public DataType UnifyPointer(Pointer ptrA, DataType b)
+        private bool IsNestedStructure(
+            StructureField fa, StructureField fb,
+            out StructureField fNestedStruct, out StructureField fOther)
+        {
+            var strFa = fa.DataType.TypeReferenceAs<StructureType>();
```

Comment:
What about moving `IsNestedStructure` and `MergeFieldIntoNestedStructure` to new `NestedStructureUnifier` class? And so `fNestedStruct`, `fOther` become internal fields and shouldn't be returned.